node/node.hpp

```diff
@@ -46,6 +49,10 @@ namespace bzn
 
         void send_message_str(const boost::asio::ip::tcp::endpoint& ep, std::shared_ptr<bzn::encoded_message> msg, bool close_session) override;
 
+        void send_message(const bzn::uuid_t &uuid, std::shared_ptr<bzn_envelope> msg, bool close_session) override;
+
+        void set_consensus(const std::shared_ptr<bzn::pbft_base>& pbft) override;
```

Comment:
Weren't we going to add pbft to start and remove this method?

Reply:
No. We discussed it, but I thought we decided to leave the code the way it was.